chainerrl/wrappers/atari_wrappers.py

```diff
@@ -10,6 +10,8 @@
 
 from gym import spaces
 
+import chainerrl
```

Comment:
Since you're changing the file, perhaps we should change the header from "This file is a fork from a MIT-licensed project" to "This file _adapted_ from an MIT-licensed project..."

Reply:
"fork" means it already has changes, no?